### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by vichanskyLab**

### CODE REVIEW

File: R/Descriptive_Statistics.Rmd [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+library(tinytex)
+library(ggplot2)
+library(scales)
+library(wesanderson)
+
+```
+Import data and check data format.
+```{r 1}
+
+data <- read.csv("30-11-2020.csv", header = TRUE)
```
Comment (roelhem) [inline R/Descriptive_Statistics.Rmd]: Are you sure that this link isn't broken in this way?

Shouldn't `30-11-2020.csv` be `../data/30-11-2020.csv`?

Comment (vichanskyLab) [reply R/Descriptive_Statistics.Rmd]: Yes okay, last minute change of putting into the 'data' folder does that. I will push again, one sec

Comment (roelhem) [reply R/Descriptive_Statistics.Rmd]: Aaah! I see! It is gets the data from a zip-file.

I'll write a quick setup function to make it work tomorrow!

Comment (roelhem) [reply R/Descriptive_Statistics.Rmd]: ^^ Never mind, let's see if this wil work!

Comment (roelhem) [reply R/Descriptive_Statistics.Rmd]: Fixed this!

**Review by roelhem — COMMENTED** [review]
All OK, except for the fact that Oct-19 should be interpreted as 10-19

**Pull request merged**